content/tutorials/quickstart/py-create-conditional-escrows.md

```diff
@@ -0,0 +1,625 @@
+  - Transaction Sending
+  - XRP
+---
+# Create Conditional Escrows Using Python
+
+This example shows how to:
+
+1. Create escrow payments that become available when any account enters a fulfillment code.
+
+2. Complete a conditional escrow transaction.
+
+3. Cancel a conditional escrow transaction.
+
+[![Conditional Escrow Tester Form](img/quickstart-py-conditional-escrow-1.png)](img/quickstart-py-conditional-escrow-1.png)
+
+
+## Prerequisites
+
+Download and expand the [Quickstart Samples](https://github.com/XRPLF/xrpl-dev-portal/tree/master/content/_code-samples/quickstart/py/){.github-code-download} archive.
+
+## Usage
+
+### Create Escrow
+
+You  create a condition-based escrow using a fulfillment code associated with a condition code. Create the condition/fulfillment pair using the `five-bells-condition` application.
+
+Install `five-bells-condition`:
+
+1. In a terminal window, navigate to your local `Quickstart` directory (for convenience).
+2. Enter the command `npm install five-bells-condition`.
+
+To create a condition/fulfillment pair:
+
+1. In a terminal window, navigate to your `Quickstart` directory.
+2. Enter the command `node getConditionAndFulfillment.js`.
+3. Copy and save the generated Condition and Fulfillment pair.
+
+[![Condition and Fulfillment](img/conditional-escrow2.png)](img/conditional-escrow2.png)
+
+To get test accounts:
+ 
+1. Open and run `lesson9-conditional-escrow.py`.
+2. Get test accounts.
+    1. If you have existing account seeds
+        1. Paste Standby account seed in the **Standby Seed** field.
+        2. Click **Get Standby Account**.
+        3. Click **Get Standby Account Info**.
+        4. Paste Operational account seed in the **Operational Seed** field.
+        5. Click **Get Operational Account**.
+        6. Click **Get Op Account Info**.
+    2. If you do not have account seeds:
+        1. Click **Get Standby Account**.
+        2. Click **Get Standby Account Info**.
+        3. Click **Get Operational Account**.
+        4. Click **Get Op Account Info**.
+
+[![Escrow Tester with Account Information](img/quickstart-py-conditional-escrow-2.png)](img/quickstart-py-conditional-escrow-2.png)
+
+### Create Conditional Escrow:
+
+<div align="center">
+<iframe width="560" height="315" src="https://www.youtube.com/embed/nWaMAr84qR0?si=LkF-vwQ3QJ4XNhJA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+</div>
+
+When you create a conditional escrow, you need to specify the `Condition` value you generated above. You must also set a cancel date and time, after which the escrow is no longer available.  
+
+To create a conditional escrow:
+
+1. Enter an **Amount** to transfer.
+2. Copy the **Operational Account** value.
+3. Paste it in the **Destination Account** field.
+4. Enter the **Escrow Condition** value.
+5. Enter the **Escrow Cancel (seconds)** value.
```

Comment:
Does your code convert to a Ripple epoch value?

Reply:
Yes.